ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -45,3 +123,53 @@ def test_numpy_bincount(
         weights=None,
         minlength=0,
     )
+
+
+@handle_frontend_test(fn_tree="numpy.histogram_bin_edges", data=_histogram_helper())
+def test_numpy_histogram_bin_edges(
```

Comment:
Part of tests are failing due to bfloat16 is not supported by numpy. Either we should not test for bfloat16 dtype (avoiding it when generating arrays for testing) or handle it explicitly